recipe/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "peft" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/huggingface/peft/archive/v{{ version }}.tar.gz
+  sha256: 954b86dcf3dad64562ce7b9167485ac42c47fcc668d82ea99874001aa585f53e
+
+build:
+  skip: True  # [s390x or ppc64le]
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --no-build-isolation . -vv
+
+
+requirements:
+  host:
+    - python
+    - pip
+    - wheel
+    - setuptools
+  run:
+    - python
+    - numpy >=1.17
+    - packaging >=20.0
+    - psutil
+    - pyyaml
+    - pytorch >=1.13.0
+    - transformers
+    - tqdm
+    - huggingface_accelerate >=0.21.0
```

Comment:
where is this requirement coming from? does it have anything to do with [this issue](https://github.com/huggingface/peft/issues/1066)?

Reply:
https://github.com/huggingface/peft/blob/0ae52fece17a3514116815984444116b75d9c5ca/examples/lora_dreambooth/requirements.txt#L2
we have name conflict so in conda we name it huggingface_accelerate instead of accelerate